app/build.gradle

```diff
@@ -55,6 +55,7 @@ dependencies {
     ktlint "com.github.shyiko:ktlint:0.31.0"
 
     implementation project(':core')
+    implementation project(':test_shared')
```

Comment:
The test_shared module should just be put into core_dependencies.gradle as both a testImplementation and androidTestImplementation.